### PR TITLE
[NET-7787] Update JWT docs for APIGateway

### PR DESCRIPTION
**Pull request by jm96441n**
### Description

Updates the JWT docs for APIGateway with usage examples

### PR Checklist

* [ ] updated test coverage
* [x] external facing docs updated
* [ ] appropriate backport labels added
* [ ] not a security concern

### CODE REVIEW

File: website/content/docs/connect/gateways/api-gateway/secure-traffic/verify-jwts-k8s.mdx [hunk 1]
```diff
@@ -55,20 +126,84 @@ Create an `RouteAuthFilter` values file and configure the following fields. Refe
...
+      backendRefs:
+        - kind: Service
+          name: user-service
+          port: 8081
+```
+
 ## Apply the configurations
 
-Run the `kubectl apply` command and specify the values files to apply the configurations. The following example applies the values files stored in the `jwt-routes` directory:
+Run the `kubectl apply` command and specify the values files to apply the configurations. The following example applies the above values files stored in a directory named `jwt-routes`:
```
[the remark this answers is not in the document]
Comment (jm96441n) [reply website/content/docs/connect/gateways/api-gateway/secure-traffic/verify-jwts-k8s.mdx:193]: yeah, I was trying to keep this usage in line with the original docs (the reference to the `jwt-routes` directory didn't really make sense on a first pass so I tried to re-word it to make it make a bit more sense)

File: website/content/docs/connect/gateways/api-gateway/secure-traffic/verify-jwts-k8s.mdx [hunk 2]
```diff
@@ -55,20 +126,84 @@ Create an `RouteAuthFilter` values file and configure the following fields. Refe
...
+```yaml
+// route-auth-filter.yaml
+apiVersion: consul.hashicorp.com/v1alpha1
+kind: RouteAuthFilter
+metadata:
+  name: auth-filter
+spec:
+  jwt:
+    providers:
+    - name: "local"
```
[the remark this answers is not in the document]
Comment (jm96441n) [reply website/content/docs/connect/gateways/api-gateway/secure-traffic/verify-jwts-k8s.mdx:143]: ah this was a typo

File: website/content/docs/connect/gateways/api-gateway/secure-traffic/verify-jwts-k8s.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (jm96441n) [reply website/content/docs/connect/gateways/api-gateway/secure-traffic/verify-jwts-k8s.mdx:204]: honestly I'm okay removing this section entirely, it read weird to me on my first pass

**Comment by david-yu** [comment]
Is it possible to get this somehow pushed through? Thanks for working on this!

**Comment by hc-github-team-consul-core** [comment]
@jm96441n, a backport is missing for this PR [20800] for versions [1.15,1.16,1.17] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@jm96441n, a backport is missing for this PR [20800] for versions [1.15,1.16,1.17] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@jm96441n, a backport is missing for this PR [20800] for versions [1.15,1.16,1.17] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@jm96441n, a backport is missing for this PR [20800] for versions [1.15,1.16,1.17] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@jm96441n, a backport is missing for this PR [20800] for versions [1.15,1.16,1.17] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@jm96441n, a backport is missing for this PR [20800] for versions [1.15,1.16,1.17] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@jm96441n, a backport is missing for this PR [20800] for versions [1.15,1.16,1.17] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@jm96441n, a backport is missing for this PR [20800] for versions [1.15,1.16,1.17] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@jm96441n, a backport is missing for this PR [20800] for versions [1.15,1.16,1.17] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@jm96441n, a backport is missing for this PR [20800] for versions [1.15,1.16,1.17] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@jm96441n, a backport is missing for this PR [20800] for versions [1.15,1.16,1.17] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@jm96441n, a backport is missing for this PR [20800] for versions [1.15,1.16,1.17] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@jm96441n, a backport is missing for this PR [20800] for versions [1.15,1.17,1.18] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@jm96441n, a backport is missing for this PR [20800] for versions [1.15,1.17,1.18] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@jm96441n, a backport is missing for this PR [20800] for versions [1.15,1.17,1.18] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@jm96441n, a backport is missing for this PR [20800] for versions [1.15,1.17,1.18] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@jm96441n, a backport is missing for this PR [20800] for versions [1.15,1.17,1.18] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@jm96441n, a backport is missing for this PR [20800] for versions [1.15,1.17,1.18] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
@jm96441n, a backport is missing for this PR [20800] for versions [1.15,1.17,1.18] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Pull request merged**